### PR TITLE
Add --overwrite option to plugin assets copy shell task

### PR DESCRIPTION
**Pull request by andrej-griniuk**
Refs #10151

### CODE REVIEW

**Comment by dereuromark** [comment]
Looks good.

File: src/Shell/Task/AssetsTask.php [hunk 1]
```diff
@@ -112,9 +112,10 @@ protected function _list($name = null)
      *
      * @param array $plugins List of plugins to process
      * @param bool $copy Force copy mode. Default false.
+     * @param bool $overwrite Overwrite existing files.
      * @return void
      */
-    protected function _process($plugins, $copy = false)
+    protected function _process($plugins, $copy = false, $overwrite = false)
```
[the remark this answers is not in the document]
Comment (dereuromark) [reply src/Shell/Task/AssetsTask.php]: It is not true in the option parser, it is only set to boolean option, but still defaults to false.

[the remark this answers is not in the document]
Comment (markstory) [reply src/Shell/Task/AssetsTask.php]: Derp. I should've paid more attention.

**Comment by markstory** [comment]
Thanks!

**Pull request merged**